consensus/service/src/api/client_api_service.rs

```diff
@@ -218,6 +218,16 @@ impl ConsensusClientApi for ClientApiService {
     ) {
         let _timer = SVC_COUNTERS.req(&ctx);
 
+        {
+            let session = ClientSession::from(msg.channel_id.clone());
+            let mut tracker = self.tracked_sessions.lock().expect("Mutex poisoned");
+            if let Some(_session_info) = tracker.get(&session) {
+                // TODO: Update fields
+            } else {
+                // TODO: Populate new session metadata
+                tracker.put(session, ClientSessionTracking {});
```

Comment:
For future, this should use the `Default` impl, ala `ClientSessionTracking::default()`.

Reply:
Most likely I'll be initializing it with a "last touched" timestamp or something like that in here, so default() would not be ideal but also it would not be an empty initializer like this. The `ClientSessionTracking` struct is a stub deliberately for now.

Is `ClientSessionTracking{foo, bar}` preferred or should I go for more of a `ClientSessionTracking::new(foo, bar)` style?

Reply:
I generally prefer the constructor method.